### ISSUE TITLE
allow soft failure when not a git repo

### ISSUE THREAD

**Issue by kgeis** [n0mer/gradle-git-properties#27]
I have exported a git repo into another directory and I am working with it. Since it does not have git metadata, the gradle-git-properties plugin fails.

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateGitProperties'.
> org.eclipse.jgit.errors.RepositoryNotFoundException: repository not found: /Users/kgeis/temp/phoebe
```

I would like for it not to fail.

**Comment by n0mer** [comment]
fixed in 1.4.18

**Comment by robmoore** [comment]
Sorry for the basic Gradle question but how would one set this value?
